src/api/kube.ts

```diff
@@ -871,15 +896,13 @@ export class KubeHelper {
     if (!currentCluster) {
       throw new Error('Failed to get current Kubernetes cluster: returned null')
     }
-    const token = await this.getDefaultServiceAccountToken()
-
     const agent = new https.Agent({
       rejectUnauthorized: false
     })
     let endpoint = ''
     try {
       endpoint = `${currentCluster.server}/healthz`
-      let response = await axios.get(`${endpoint}`, { httpsAgent: agent, headers: { Authorization: 'bearer ' + token } })
+      let response = await axios.get(`${endpoint}`, { httpsAgent: agent })
```

Comment:
why token is removed there ?

Reply:
Otherwise, the following error appears on minishift when logged as developer
![Screenshot_20190910_135245](https://user-images.githubusercontent.com/5887312/64624567-8bf34380-d3f3-11e9-818c-d0904afa8c38.png)

Reply:
hum but if you remove it there are failures with some k8s clusters....

Reply:
I can not guarantee. 
I only discovered that it does not work on minishift when logged in as non-admin user.
And tested my changes (without token) that is works on minikube(without explicit specifying RBAC enabled) and minishift.

I can test it on like `minikube` with RBAC specified or OpenShift 4.1

Reply:
Tested on CRC 4.1 as well

Reply:
@benoitf Thanks for sharing concrete implementation where it fails.
Because on one of OSIO cluster, healthz endpoint is public and it supposed by be secure
https://api.starter-us-east-2.openshift.com/healthz
I'm going to revert it and create a separate issue that there is an issue with `healthz` if user does not have cluster admin rights.